src/plugins/auto/auto_schedule.cpp

```diff
@@ -310,7 +310,7 @@ void AutoSchedule::TryToLoadNetWork(AutoLoadContext& context, const std::string&
             int maxNumThreads = 0;
             try {
                 maxNumThreads = _autoSContext->_core->GetConfig(device, GPU_CONFIG_KEY(MAX_NUM_THREADS)).as<int>();
-            } catch (...) {
+            } catch (std::exception&) {
```

Comment:
Can we get any other exceptions expect `ov::Exception` or `InferenceEngine::Exception`?